test/CrossImport/remark-option.swift

```diff
@@ -0,0 +1,7 @@
+// This file tests that the -Rcross-import option causes an appropriate remark to be emitted
+// RUN: %empty-directory(%t)
+// RUN: cp -r %S/Inputs/lib-templates/* %t/
+// RUN: %target-typecheck-verify-swift -enable-cross-import-overlays -Rcross-import -I %t/include -I %t/lib/swift -F %t/Frameworks
+
+import UniversalExports
+import ThinLibrary // expected-remark {{Import of 'ThinLibrary' and 'AlwaysImported' triggered a cross-import of '_AlwaysImportedOverlay'}}
```

Comment:
`UniversalExports` and `ThinLibrary` form a slightly weird example (we're looking at how the remark is diagnosed when the bystanding module is re-exported); it might actually be helpful to have them in this test case, but I would first try importing `DeclaringLibrary` and `BystandingLibrary`, which have a less complicated relationship.

Also, when you import those two, import `BystandingLibrary` second and see where the remark is diagnosed. I suspect it'll be on the `BystandingLibrary` line, which is the same issue complained of in [test/CrossImport/horrible.swift](https://github.com/apple/swift/blob/829f2588b37e7c27dc09bcaae9726bb7ae5f64ee/test/CrossImport/horrible.swift#L18). The AST doesn't really have enough information about source locations to fix this issue as fully as I'd like, but it may be good to add a similar FIXME comment and perhaps update the SR to mention that the remark has the same issue as those errors.